### PR TITLE
Repository Vulnerability alert patch

### PR DESCRIPTION
**Pull request by akhilerm**
Added struct for `repository_vulnerability_alert` webhook event, as mentioned in [docs](https://developer.github.com/v3/activity/events/types/#repositoryvulnerabilityalertevent). 

Made changes where the payload is assigned and mapping done from webhook event to new struct.

### CODE REVIEW

File: github/event_types.go [hunk 1]
```diff
@@ -710,6 +710,27 @@ type RepositoryEvent struct {
 	Installation *Installation `json:"installation,omitempty"`
 }
 
+// RepositoryVulnerabilityAlertEvent is triggered when a security alert is created, dismissed, or resolved.
+//
+// GitHub API docs: https://developer.github.com/v3/activity/events/types/#repositoryvulnerabilityalertevent
+type RepositoryVulnerabilityAlertEvent struct {
+	// Action is the action that was performed. This can be: "create", "dismiss", "resolve"
```
Comment (gmlewis) [inline github/event_types.go]: nit: please add period to end of sentence for godocs.

Comment (akhilerm) [reply github/event_types.go]: period added and pushed.

**Comment by gmlewis** [comment]
Thank you, @gauntface!
Merging.

**Comment by akhilerm** [comment]
PR for issue https://github.com/google/go-github/issues/1005.

Thanks @gmlewis, @gauntface for helping me make my first PR in a Google repo.

**Pull request merged**